create-container.sh

```diff
@@ -5,15 +5,81 @@ set -e
 # Uncomment the following line to debug the script
 # set -x
 
+function print_help {
+    echo """
+    Usage: devenv [subcommand]
+
+A bash script to create and manage development environments using privileged LXC linux containers.
+
+Subcommands:
+    init - Generate the default configuration file. Create the .devenv
+
+
+More info: https://github.com/coopdevs/devenv
+    """
+}
+
+function init {
+    CONFIG="$1"
+    if [ -f "$CONFIG" ]; then
+      printf "The devenv config file already exists. Execute:\n\n$ cat .devenv\n"
+      exit 1
+    else
+      cat > "${CONFIG}" << EOF
+# File created with devenv init command
+
+NAME="<container name>"
+DISTRIBUTION="<SO distribution>"
+RELEASE="<SO release>"
```

Comment:
I would rather enforce ubuntu bionic, because that's what we mostly use. Then, when we migrate we'll open a PR to upgade it.

Reply:
Well, I wouldn't say "enforce", as this is just a default that can be changed. If this is what you meant, I agree to set some defaults. I would also add the architecture, since we are using amd64  and I would rather not step into arm before thinking it twice:
```
DISTRIBUTION="ubuntu"
RELEASE="bionic"
ARCH="amd64"
```